### PR TITLE
Add basic sign/verify metrics

### PR DESCRIPTION
**Pull request by ixjohnny**
- Add Prometheus based metrics to ads.cert library.
  - Added counters for signing and verification, errors, and result.
- Updated examples to expose prometheus metrics endpoint.

### CODE REVIEW

**Review by shahinrahbariasl — APPROVED** [review]

**Comment by ixjohnny** [comment]
Let me know if I could get one more review from one of you before merging this into main @grossjo @cmlight. Thanks!

**Review by cmlight — APPROVED** [review]

File: cmd/server/main.go
Comment (ixjohnny) [reply cmd/server/main.go:9]: Those will be coming in my next PR. Just wanted to get some feedback with basic counter examples first.

File: internal/metrics/metrics.go
Comment (ixjohnny) [reply internal/metrics/metrics.go:62]: For verification, the signatures are either valid or not valid. I'm not sure what kind of results you would expect from signing?

**Pull request merged**
